.github/workflows/checks-sealights.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Not sure, but we might need some changes here to make sure it behaves as expected for post-merge CI.

Reply:
I feel that would be taken care of by this environment variable
https://github.com/conforma/cli/blob/7bc924d7ef2bae6754b1f0d29f420620fa874aac/.github/workflows/checks-sealights.yaml#L115

in case of push events, it would use the sha of the merge commit, which is expected.